pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -86,13 +86,13 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "zed-editor";
-  version = "0.158.1";
+  version = "0.158.2";
 
   src = fetchFromGitHub {
     owner = "zed-industries";
     repo = "zed";
     rev = "refs/tags/v${version}";
-    hash = "sha256-X5FlNjAYG5W+RTdmyZHF1KYHxdwJGwk/qpOgj0JVD/c=";
+    hash = "sha256-1mNJ3uL5/Sxwiiq2fO+yE9SHiM/5FQWmnNgOEMWpU9s=";
     fetchSubmodules = true;
```

Comment:
We already removed `fetchSubmodules` in the darwin fix (which currently resides in staging-next), maybe just remove it in this PR as well as it caused merged conflicts when the update to 0.158.1 was merged. Zed does not have submodules in its repository anymore.

Reply:
Done